presenter-in-iframe/README.md

```diff
@@ -0,0 +1,6 @@
+# Presenter in an Iframe
+
+## Description
+
+An debugging tool that deploy an hard coded (fill `src` attribute of `iframe` with `id` `thisIframe` of  `index.html:60`) presenter link in an basic web page
+the "inverse" demo of the [events-tab](../events-tab/README.md) integration.
```

Comment:
```suggestion
A debugging tool that embeds a hard coded project in a web page using FieldTwin Presenter and demonstrates how to post external messages to it. This can be considered as the "inverse" demo of the [events-tab](../events-tab/README.md) integration.

To set the project, fill the `src` attribute of `iframe` with `id` `thisIframe` at `index.html:60` with a View Link created in FieldTwin Design.
```